specification/semantic_conventions.md

```diff
@@ -27,3 +28,34 @@ All Splunk distributions of OpenTelemetry,
 | Attribute  | Type | Description  | Examples  | Required |
 |---|---|---|---|---|
 | `splunk.distro.version` | string | The version number of the Splunk distribution being used. | `1.5.0` | Yes |
+
+Note: this section does not apply to Real User Monitoring libraries, as they do not use the OpenTelemetry Resource.
+
+## Real User Monitoring Spans and Attributes
+
+Real User Monitoring (RUM) libraries MUST set the `service.name` resource attribute to the value of the `applicationName`
+configuration property. This is the only resource attribute that RUM libraries are supposed to set because it's the only one
+the Zipkin exporter can understand.
+
+The following attributes MUST be added to all spans emitted by RUM libraries:
```

Comment:
What happens if they are not set?

Reply:
Good question. Do we have a default, or do we leave it out?

Reply:
Hmm, what do you mean "they are not set"?
All those attributes are either something that a RUM library must implement (session id, version, component) or something that user must provide (app).

Reply:
So there are two personas here 1) the library developer -- this is a must, but I assume is not enforced 2) the software developer using this instrumentation -- what if they do not set the required attributes?

Reply:
> 1) the library developer -- this is a must, but I assume is not enforced

Well, it is enforced by being a requirement in the GDI spec.

> 2) the software developer using this instrumentation -- what if they do not set the required attributes?

They do not have to set anything in the first place - those attributes must be added by us, the library developer, the user doesn't have to do anything to have them.